modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/footer/index.html

```diff
@@ -0,0 +1,17 @@
+<div class="cp_footer_home">
+<div class="cp_footer_container">
+	<h6 data-lfr-editable-id="cp_footer_text" data-lfr-editable-type="rich-text" class="cp_footer_text">
+	  © 2023 Liferay Inc. All Rights Reserved &nbsp; &nbsp; &nbsp; &nbsp; |
+	</h6>
+
+	<h6 class="cp_footer_text">
+			 <a href="/web/cp-footer/cookie-policy" data-lfr-editable-id="link" data-lfr-editable-type="link">
+	  Cookie policy
+	</a>
+	</h6>
+</div>
+
+<div>
+	<h6 class="cp_footer_text" data-lfr-editable-id="cp_footer_text_feedback" data-lfr-editable-type="rich-text" id="cp_footer_text_feedback">Send FeedBack</h6>
```

Comment:
"Send FeedBack" text should be "Send Feedback" with a lowercase "b"

Reply:
Done!